api/v2/clairpb/clair.proto

```diff
@@ -0,0 +1,133 @@
+syntax = "proto3";
+option go_package = "clairpb";
+
+package clairpb;
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+message Vulnerability{
```

Comment:
Nope; based on the style, I shouldn't use the camel case for fields: https://developers.google.com/protocol-buffers/docs/style